xml/Makefile.am

```diff
@@ -145,7 +145,16 @@ api/api-result-%.rng: $(API_files) Makefile.am
 	echo '      </optional>' >> $@
 	echo '      <element name="status">' >> $@
 	echo '        <attribute name="code"> <data type="integer" /> </attribute>' >> $@
-	echo '        <text />' >> $@
+	echo '        <attribute name="message"> <text /> </attribute>' >> $@
+	echo '        <optional>' >> $@
+	echo '          <element name="errors">' >> $@
+	echo '            <oneOrMore>' >> $@
+	echo '              <element name="error">' >> $@
+	echo '                <text />' >> $@
+	echo '              </element>' >> $@
+	echo '            </oneOrMore>' >> $@
+	echo '          </element>' >> $@
+	echo '        </optional>' >> $@
```

Comment:
Interestingly, this technically should require a bump in the schema version to 2.0, but since it's not actually in a file we can't do that :). But I'm guessing we'll have other schema changes before the next release, so we can handle it then. Hopefully we can declare it 2.0 and move it from experimental to supported at that time.

Reply:
I was surprised to discover that too.  It looks like the basic strategy for anything generated in the Makefile should be to put as much as possible into subfiles and just hope that the basic structure never needs to change.